src/components/ImageView/index.js

```diff
@@ -84,8 +86,13 @@ class ImageView extends PureComponent {
             imgRight = imgLeft + (fitRate * width);
         }
 
+        //In case image loading is delayed than onLayout callback of the root View caused internet speed
```

Comment:
Actually, my internet speed is not good sometimes. And I noticed that it has some issue because we would not load image correctly.
If you don't need this, I will remove.